tools/tests/create_new_yml_test.py

```diff
@@ -0,0 +1,36 @@
+# Given XLSX_1 When no mentors Then create new Yml
+# Given XLSX_1 When existent yml Then Merge Yml's
+# Given XLSX_2 When existent yml Then Merge Yml
+# Given XLSX_2 When All mentors are present Then no change happen
+
+import tempfile
+import os
+import sys
+import pytest
+from file_utils import TOOLS_PATH
+from automation import run_automation, read_yml_file, WriteMode
+
+MENTOR_ADRIANA = "Adriana Zencke Zimmermann"
+MENTOR_A = "Mentor A"
+MENTOR_B = "Mentor B"
+MENTOR_C = "Mentor C"
+
+def test_write_mentors_skip_zero_rows(monkeypatch):
```

Comment:
You don't need to create a single file for unit test, you can create one test file with the same name as original source, but with _test only it is too big you need to split, or per domain as we discussed.